frontend/src/modules/LoadTestContentRedux.js

```diff
@@ -0,0 +1,39 @@
+// Action Types
+const UPDATE_TEST_META_DATA = "emibInbox/UPDATE_TEST_META_DATA";
+
+// Action Creators
+const updateTestMetaDataState = testMetaData => ({ type: UPDATE_TEST_META_DATA, testMetaData });
+
+const getTestMetaData = testName => {
+  return async function() {
+    let metaDataContent = await fetch(`/api/test-meta-data/?test_name=${testName}`, {
+      method: "GET",
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      cache: "default"
+    });
+    return await metaDataContent.json();
+  };
+};
+
+// Initial State
+const initialState = {
+  testMetaData: ""
```

Comment:
shouldn't this be: `testMetaData: {}` if you're getting a json object?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/354/commits/1350782bbcd9879f8746bb8a91fcefd1a45d388e